Item/Item.module.less

```diff
@@ -44,10 +44,6 @@
 		color: @agate-item-color;
 		background-color: @agate-item-bg-color;
 
-		&:last-child {
-			border-bottom-color: transparent;
-		}
-
```

Comment:
Hello @MikyungKim , @jeonghee27 

There is no bottom border in the last item in our requirement.
We supported it before with this line, but now it didn't work because VirtualList wrapped the item with an additional div element.

In this PR, it is supported in the sampler. But do you have any idea to support it in the Item or the VirtualList/Scroller component?

Reply:
Because we don't have any requirement about this, I think current direction is not bad. 
and I have one question, would it be okay to have no borders in the `VirtualGridList` sampler?